.github/workflows/build.yml

```diff
@@ -97,6 +97,20 @@ jobs:
           ls -lah coverage.info
           cp coverage.info ../
 
+  badge_build_cpu_sdk:
```

Comment:
this should be removed from this pr and put in another PR

Reply:
OK, i will handle that later.